src/frontend/replayserver.cc

```diff
@@ -135,7 +136,16 @@ int main( void )
         }
 
         if ( best_score > 0 ) { /* give client the best match */
-            cout << HTTPResponse( best_match.response() ).str();
+            string best_response = HTTPResponse( best_match.response() ).str();
+            // Remove first line if status line is first
+            int pos = best_response.find( "\n" );
+            string first_line = best_response.substr( 0, pos );
+            if ( first_line.find( "HTTP" ) == 0 ) {
+                int start_pos = first_line.find( ' ' );
+                cout << "Status: " << first_line.substr( start_pos+1, 3 );
```

Comment:
Ok, but we would need to do this consistently then -- e.g. line 151 and 157 where replayserver also prints an HTTP response.

In general, rather than serialize the HTTPResponse object (`.str()`) and then try to parse the serialized output (looking for an HTTP status code and parsing it out), better would probably be to have a different serialization method that just serializes it the way we want.